components/massmint/Massmint.vue

```diff
@@ -0,0 +1,146 @@
+        <o-icon pack="fas" icon="arrow-left" size="small" class="mr-1" />
+        {{ $t('massmint.backToOnbaording') }}
+      </NeoButton>
+      <div class="is-flex">
+        <TabItem
+          v-for="tab in tabs"
+          :key="tab"
+          :active="tab === 'Mass Mint'"
+          :text="tab"
+          :to="route.path"
+          class="mobile-width" />
+      </div>
+    </section>
+    <hr class="m-0" />
+    <section
+      class="pt-6 is-flex is-flex-direction-column is-align-items-center">
+      <p class="mb-4">{{ $t('massmint.chooseCollection') }}</p>
+      <NeoDropdown
+        class="dropdown-width"
+        :disabled="!isLogIn"
+        @active-change="toggleOpen">
+        <NeoButton
+          v-if="!selectedCollection"
+          class="dropdown-width"
+          :label="$t('massmint.selectCollection')"
+          :icon="isDropdownopen ? 'chevron-up' : 'chevron-down'" />
+        <NeoButton
+          v-else
+          class="dropdown-width"
+          :icon="isDropdownopen ? 'chevron-up' : 'chevron-down'">
+          {{ selectedCollection.name || selectedCollection.id }}
+          <o-icon
+            v-if="selectedCollection"
+            icon="check"
+            size="small"
+            variant="success"
+            class="ml-3" />
+        </NeoButton>
+        <template v-if="collectionsEntites?.length" #items>
+          <NeoDropdownItem
+            v-for="collection in collectionsEntites"
+            :key="collection.id"
+            class="dropdown-width"
+            @click.native="selectCollection(collection)">
+            <span class="px-5">
+              {{ collection.name || collection.id }}
+            </span>
+          </NeoDropdownItem>
+          <NeoDropdownItem class="dropdown-width" has-link>
+            <nuxt-link :to="`/${urlPrefix}/create`" class="w-100">
+              <div class="w-100">
+                <o-icon pack="fas" icon="plus" size="small" class="mr-1">
+                </o-icon>
+                {{ $t('massmint.createNewCollection') }}
+              </div>
+            </nuxt-link>
+          </NeoDropdownItem>
+        </template>
+        <template v-else #items>
+          <NeoDropdownItem disabled class="dropdown-width">
+            <span class="px-5"> {{ $t('massmint.noCollection') }} </span>
+          </NeoDropdownItem>
+          <NeoDropdownItem class="dropdown-width">
+            <div class="w-100">
+              <o-icon pack="fas" icon="plus" size="small" class="mr-1">
+              </o-icon>
+              {{ $t('massmint.createNewCollection') }}
+            </div>
+          </NeoDropdownItem>
+        </template>
+      </NeoDropdown>
+    </section>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { NeoButton, NeoDropdown, NeoDropdownItem } from '@kodadot1/brick'
+import { OIcon } from '@oruga-ui/oruga'
+import { useMassmintsStore } from '@/stores/massmint'
+import TabItem from '@/components/explore/tab/TabItem.vue'
```

Comment:
How about moving `TabItem` component to `components/shared/TabItem.vue`

Reply:
done